api/v2/Routes/students/grades/index.js

```diff
@@ -8,18 +8,30 @@ import { isAdmin } from '../../../../lib/userlib.mjs';
 const router = Router({ mergeParams: true });
 
 router.get('/', async (req, res) => {
-    const { id } = req.params;
-    const maxScores = await getMaxScores();
+    const { id } = req.params; // the id is the student's email
     let studentScores;
```

Comment:
Move this inside the try block if not otherwise referenced in this scope.

Reply:
resolved in [f6beb91](https://github.com/AFA-Tooling/GradeView/pull/12/commits/f6beb91531116dbd924697e08961e66d79b0e17a)